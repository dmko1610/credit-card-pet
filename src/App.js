import React from "react";
import { StatusBar } from "react-native";
import { MainScreen } from "./screens/MainScreen";

const App = () => {
  return (
    <>
      <StatusBar barStyle="dark-content" backgroundColor="white" />
      <MainScreen />
    </>
  );
};

export default App;
