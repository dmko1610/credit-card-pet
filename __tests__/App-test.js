import "react-native";
import React from "react";
import { render } from "react-native-testing-library";
import renderer from "react-test-renderer";
import App from "../src/App";

describe("Credit Card App", () => {
  it("renders the application", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
