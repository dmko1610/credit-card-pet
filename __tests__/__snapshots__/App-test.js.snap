// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credit Card App renders the application 1`] = `
<View
  style={Object {}}
>
  <View
    style={
      Object {
        "padding": 20,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "green",
          "borderRadius": 15,
          "height": 200,
          "width": 710,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
            "margin": 10,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
          Some Icon
        </Text>
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
          Bank Logo
        </Text>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "alignSelf": "center",
              "color": "white",
            }
          }
        >
          5500 4000 1233 1233
        </Text>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
          Card Holder
        </Text>
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
          Expires
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
          John Wick
        </Text>
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
          13/12
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "gray",
        "borderRadius": 12,
        "borderWidth": 1,
        "elevation": 4,
        "height": 400,
        "margin": 10,
        "position": "absolute",
        "top": 150,
        "width": 730,
        "zIndex": 0,
      }
    }
  />
</View>
`;
